libs/platform/src/lib/components/form/checkbox/checkbox.module.ts

```diff
@@ -0,0 +1,12 @@
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { CheckboxModule, FormModule as FdFormModule } from '@fundamental-ngx/core';
+import { CheckboxComponent } from './checkbox.component';
+
+@NgModule({
+    declarations: [CheckboxComponent],
+    imports: [CommonModule, FormsModule, ReactiveFormsModule, FdFormModule, CheckboxModule],
```

Comment:
We should avoid using FormModule from core. This is big problem. Even using checkbox FormModule is importing all the other modules which we dont need. Its the same as you would import our big PlatformModule that has everything.

Reply:
removed unused ReactiveFormModule and core FormModule.